<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BezierMainWindow</class>
 <widget class="QWidget" name="BezierMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bezier Patch</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="BezierWidget" name="bezierWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="useOpenCL">
       <property name="text">
        <string>Use OpenCL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="localWorkSize">
       <property name="title">
        <string>Local Work Size</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QRadioButton" name="localWorkSizeDefault">
          <property name="text">
           <string>Default</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="localWorkSize1x1">
          <property name="text">
           <string>1x1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="localWorkSize1x8">
          <property name="text">
           <string>1x8</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="localWorkSize8x1">
          <property name="text">
           <string>8x1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="localWorkSize8x8">
          <property name="text">
           <string>8x8</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="localWorkSize16x16">
          <property name="text">
           <string>16x16</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BezierWidget</class>
   <extends>QWidget</extends>
   <header>bezierwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
